src/app/lib/access/logger/custom_file_output.dart

```diff
@@ -0,0 +1,50 @@
+import 'dart:convert';
+import 'dart:io';
+
+import 'package:logger/logger.dart';
+
+/// Custom implementation of [LogOutput] to write the log output into a file.
+final class CustomFileOutput extends LogOutput {
+  final File file;
+  final bool overrideExisting;
+  final Encoding encoding;
+
+  /// Regex used to remove ansi color codes from from log files.
+  final ansiPattern = RegExp(r'\x1B\[\d+(;\d+)*m');
+
+  IOSink? _sink;
```

Comment:
Should probably add a comment to explain what this is? It's not clear right away.